src/dev-site/index.ts

```diff
@@ -1,4 +0,0 @@
-import "./global-definitions";
-import "./html-node";
-import "./theme";
-import "./renderer";
```

Comment:
Why is this not listed as renamed, src/devSite/index.ts, when src/dev-site/data.ts is renamed to src/devSite/data.ts?

Reply:
Most likely because
```
import "./global-definitions";
import "./html-node";
import "./theme";
import "./renderer";
```
was changed to
```
import "./globalDefinitions";
import "./htmlNode";
import "./theme";
import "./renderer";
```

Could seperate the commits, but as it's 4 lines it doesn't seem that important.

Reply:
I agree with you in this case, but it is important to make sure that this is not happening when there is a larger file involved. It will be more difficult to see code changes when it is listed as a new file instead of a changed file. I am not sure whether it will happen to large file or not, but if it's due to the percentage of the file that changes, it probably will not happen with a larger file

Reply:
Depending on the changes it might happen to bigger files.
If you look at [src/devSite/htmlNode.ts](https://github.com/gapitio/htmlgraphics-svg-bundler-template/pull/18/files#diff-b35cc4ca1ce580022a0b6ed96709490e9e7636db77aaccd1d2f2b94584271042)

Reply:
src/devSite/htmlNode.ts looks fine to me. It is listed as renamed. The only difference I see between those file is the percentage of the file that have been changed after the renaming. The changes that are done is similar, but if we look at the percentage lines changes it is a difference. In src/devSite/index.ts 2 out of 4 lines are changed(50%) and in src/devSite/htmlNode.ts 2 out of 10 lines are changed(20%). Is this the cause of this happening or is it related to the actual changes and placement of these changes in the file?

Reply:
Most likely

Reply:
I think the easiest way to avoid this in a large file is to not change code, that are not imports, during a file renaming process and just accept that small files that are mainly imports get listed as new files. This should not be a big problem if there isn't other code changes happening at the same time